### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by akash-1318**

### CODE REVIEW

File: components/Avatar.html [hunk 1]
```diff
@@ -36,7 +36,91 @@
...
+                            <p class="sm__text-avatar">AS</p>
+                        </div>
+                        <div class="avatar avatar-xs-size avatar-item primary-avatar">
+                            <p class="xs__text-avatar">AS</p>
+                        </div>
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
+                        src="https://carbon.now.sh/embed/2NWJU2SfYl1VRREjpqVz"
+                        style="width: 833px; height: 402px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (devdeadviz) [inline components/Avatar.html:93]: avoid inline styles

File: components/Badge.html [hunk 2]
```diff
@@ -36,7 +36,75 @@
...
+                                <img src="../assets/images/image1.jpg" alt="avatar" class="avatar_image"/>
+                            </div>
+                            <div class="status_badge"></div>
+                        </div>
+                        
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
+                        src="https://carbon.now.sh/embed/OMeKiYyWLOIs1efm1RD1"
+                        style="width: 680px; height: 618px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (devdeadviz) [inline components/Badge.html:75]: same here

File: components/css/componentStyle/Badge.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    border-radius: 50%;
+    background-color: tomato;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem;
+    color: white;
+    position: absolute;
+    top: -1rem;
+    left: -1rem;
```
Comment (devdeadviz) [inline components/css/componentStyle/Badge.css:38]: avoid repetition of code, if you require the same code in multiple classes put that code in some util class and use it wherever you need it.

File: components/css/componentStyle/Badge.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    border-radius: 50%;
+    background-color: #3f51b5;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem;
+    color: white;
+    position: absolute;
+    top: -1rem;
+    left: 11rem;
```
Comment (devdeadviz) [inline components/css/componentStyle/Badge.css]: same comes here

File: components/css/componentStyle/Badge.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    border-radius: 50%;
+    background-color: tomato;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5rem;
+    color: white;
+    position: absolute;
+    top: -1rem;
+    right: -1rem;
```
Comment (devdeadviz) [inline components/css/componentStyle/Badge.css]: same here also

**Review by devdeadviz — COMMENTED** [review]
Added some suggestions, have a look at it.

File: components/Avatar.html (hunk 1 above)
Comment (akash-1318) [reply components/Avatar.html:93]: Kuldeep that's the code snippet link i'm copying from Carbon so this inline style is included in that

File: components/css/componentStyle/Badge.css (hunk 3 above)
Comment (akash-1318) [reply components/css/componentStyle/Badge.css:38]: i will improve this

**Comment by ivinayakg** [comment]
the code looks good but you can improve on the simplicity of the code and also I felt like there was a lot of reluctant code which can be defined one time only, to reduce complicated specificity in the future, Go ahead and merge it and make the changes later as they are optimizing ones, not any functioning change.

**Review by ivinayakg — APPROVED** [review]
the code looks good but you can improve on the simplicity of the code and also I felt like there was a lot of reluctant code which can be defined one time only, to reduce complicated specificity in the future, Go ahead and merge it and make the changes later as they are optimizing ones, not any functioning change.

**Pull request merged**